buildSrc/src/main/groovy/org/elasticsearch/hadoop/gradle/BuildPlugin.groovy

```diff
@@ -64,6 +69,47 @@ class BuildPlugin implements Plugin<Project>  {
             println '==================================='
         }
     }
+
+    private static void configureBuildInfo(Project project) {
+        // Make sure the global build info plugin is applied to the root project first and foremost
+        // FIXHERE: Make a proper random test seed?
+        project.rootProject.ext.testSeed = "DEADBEEF"
+        project.rootProject.pluginManager.apply(GlobalBuildInfoPlugin.class)
+
+        // Hack new defaults into Global build info
+        if (!project.rootProject.ext.has('buildInfoConfigured')) {
+
+            JavaVersion minimumRuntimeVersion = JavaVersion.toVersion(Resources.getResourceContents("/minimumRuntimeVersion"))
+
+            // We snap the runtime to java 8 since Hadoop needs to see some significant
+            // upgrades to support any runtime higher than that
+            // Todo: Should this be provided via the ENV like the existing $RUNTIME_JAVA_HOME that we're overriding?
+            List<JavaHome> javaVersions = project.rootProject.ext.javaVersions as List<JavaHome>
+            JavaHome esHadoopRuntimeJava = javaVersions.find { it.version == 8 }
```

Comment:
FWIW, in the ES project we allow setting the `RUNTIME_JAVA_HOME` env var to do this. This might be preferable as you don't have to change the env var name itself as you bump stuff.